src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -114,14 +124,36 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
         preferences.getImportExportPreferences().setExportWorkingDirectory(file.getParent());
 
         final List<BibEntry> finEntries = entries;
+
         BackgroundTask
                 .wrap(() -> {
                     format.export(stateManager.getActiveDatabase().get(),
                             file,
                             finEntries);
                     return null; // can not use BackgroundTask.wrap(Runnable) because Runnable.run() can't throw Exceptions
                 })
-                .onSuccess(x -> dialogService.notify(Localization.lang("%0 export successful", format.getName())))
+                .onSuccess(
+                        save -> {
+                            NotificationPane notificationPane = jabRefFrame.getNotificationPane();
+                            Theme.Type themeType = preferences.getAppearancePreferences().getTheme().getType();
+                            switch (themeType) {
+                                case EMBEDDED -> notificationPane.getStyleClass().add(NotificationPane.STYLE_CLASS_DARK);
+                                case CUSTOM -> notificationPane.getStyleClass().add(notificationPane.getUserAgentStylesheet());
+                                case DEFAULT -> notificationPane.getStyleClass().removeAll(NotificationPane.STYLE_CLASS_DARK,
+                                        notificationPane.getUserAgentStylesheet());
+                            }
+                            jabRefFrame.showNotificationPane(
+                                    Localization.lang("Press \"Open\" to reveal the folder holding the saved file."),
+                                    Localization.lang("Open"),
+                                    e -> {
+                                        try {
+                                            JabRefDesktop.openFolderAndSelectFile(file, preferences);
+                                            notificationPane.hide();
+                                        } catch (IOException ioException) {
+                                            LOGGER.info(Localization.lang("Error occurs when open the folder."));
```

Comment:
Better wording: Could not open export folder
and pass the exception as second parameter to the logger.